### PR TITLE
Solved#11224

### PR DESCRIPTION
**Pull request by Swastik-Swarup-Dash**
Removed the virt_launcher.process custom SELinux policy #11224 
### What this PR does
Before this PR: we had the install policy which was not required

After this PR:we had removed install policy and any reference related to it as suggested

Fixes #removed #11224 from line 133 to 166

### CODE REVIEW

**Review by 0xFelix — COMMENTED** [review]
There looks to be overlap with https://github.com/kubevirt/kubevirt/pull/11434. Please drop the unrelated changes to the `vms-admitter.go`.

File: pkg/virt-handler/selinux/labels.go [hunk 1]
```diff
@@ -130,43 +130,7 @@ func (se *SELinuxImpl) selinux(args ...string) (out []byte, err error) {
 	return se.execFunc(virt_chroot.GetChrootBinaryPath(), argsArray...)
 }
 
-func defaultCopyPolicyFunc(policyName string, dir string) (err error) {
```
Comment (xpivarc) [inline pkg/virt-handler/selinux/labels.go:133]: This is still used or not?

**Comment by xpivarc** [comment]
/retest 
pull-kubevirt-build

**Comment by 0xFelix** [comment]
Please run `make format`, drop the unrelated changes and please make sure your commit has a meaningful title and meaningful message.

**Pull request closed without merging**